src/System.Data.Common/src/System/Xml/XPathNodePointer.cs

```diff
@@ -1619,7 +1619,7 @@ internal bool MoveToNextNamespace(XPathNamespaceScope namespaceScope)
                     node = node.ParentNode;
                 } while (node != null && node.NodeType != XmlNodeType.Element);
             }
-            // till now, node should be the next accesstor (bound) element of the element parent of current namespace node (attribute or data column)
+            // till now, node should be the next accessor (bound) element of the element parent of current namespace node (attribute or data column)
```

Comment:
is this supposed to be ancestor?

Reply:
Sure looks like it! I'll update the PR! Thanks @danmosemsft